src/Uno.UI/UI/Xaml/Media/IImageSource.wasm.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Windows.UI.Xaml.Media
+{
+	internal interface IImageSource
```

Comment:
Why did you use an interface for that ? All image sources are inheriting from `ImageSource`